assets/dns/daemonset.yaml

```diff
@@ -44,14 +44,64 @@ spec:
           timeoutSeconds: 5
           successThreshold: 1
           failureThreshold: 5
+      - name: dns-node-resolver
+        # image is set at runtime
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: hosts-file
+          mountPath: /etc/hosts
+        # env NAMESERVER and CLUSTER_DOMAIN are set at runtime
+        command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -euo pipefail
+
+          REGISTRY_SERVICE="docker-registry.default.svc"
```

Comment:
I wonder if we should make this a space separated list in case we find we need to add some services here late in the game?

Then just loop over them to build the entries.  And set it in the env section rather than inline.  We can default this here though if the env is not set.

Reply:
done